src/OpenTK.NT/Native/User32/Mouse.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Runtime.InteropServices;
+
+using HWND = System.IntPtr;
+
+namespace OpenTK.NT.Native
+{
+    public static partial class User32
+    {
+        public static class Mouse
+        {
+            [DllImport("user32.dll")]
+            public static extern HWND GetCapture();
+
+            [DllImport("user32.dll")]
+            public static extern HWND SetCapture(HWND hWnd);
+
+            [DllImport("user32.dll", SetLastError = true)]
+            [return: MarshalAs(UnmanagedType.Bool)]
+            public static extern bool ReleaseCapture();
+
+            [DllImport("user32.dll", SetLastError = true)]
+            [return: MarshalAs(UnmanagedType.Bool)]
+            public static extern bool TrackMouseEvent([In] [Out] ref TrackMouseEvent lpEventTrack);
+
+            /// <summary>
+            /// Retrieves a history of up to 64 previous coordinates of the mouse or pen.
+            /// </summary>
+            /// <param name="cbSize">The size, in bytes, of the MouseMovePoint structure.</param>
+            /// <param name="pointsIn">
+            /// A pointer to a MOUSEMOVEPOINT structure containing valid mouse
```

Comment:
Reference to define (MOUSEMOVEPOINT) should be replaced by equivalent XML doc reference (`<see>`).